broker/plugins/chronos/plugin.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import time
+# import time
```

Comment:
If this is not being used, do not comment it, remove it.

Reply:
Done!